classes/user/DAO.php

```diff
@@ -350,4 +351,18 @@ public function deleteUnvalidatedExpiredUsers(Carbon $dateTillValid, array $excl
 
         return $users->count();
     }
+
+    /**
+     * Get publication ids that have a matching setting
+     */
+    public function getIdsBySetting(string $settingName, $settingValue): Enumerable
```

Comment:
Why not use the existing functions instead...
```php
$repo = Repo::user();
$repo->getIds($repo->getCollector()->filterBySettings(['settingName' => 'settingValue']));
```
(^ that's just off the top of my head, I might've gotten details a bit wrong)